### DISCUSSION TITLE
Proportion data for SCRuB

### DISCUSSION THREAD

**Discussion by earlylifemicro-lab** [Shenhav-and-Korem-labs/SCRuB#8] [General]
Hi,
Thanks for the tool. Apart from the count matrix, may I ask whether the proportion data with unit of % (like the output from MetaPhlAn) can be decontaminated by SCRuB? If yes, how to use it, just like count matrix?
Thanks
Wang

**Comment by gaustin15** [comment]
Hi @Biofarmer; yes, you can run SCRuB using proportion data. Doing this would require an input in the same format as the `n_samples x n_features` count matrix used in SCRuB’s demos (https://shenhav-and-korem-labs.github.io/SCRuB/R-tutorial.html), where we would have floats instead of integers (with each row adding up to `1`, or `100` if we’re dealing with %s).

One small caveat is that using relative abundances/proportion data gives all samples the same power to impact SCRuB’s parameters ; while this assumption can be reasonable, it does deviate slightly from most of the validations we ran on SCRuB. So, while you can use proportion data if other options aren't available, we generally would recommend to use raw counts.

**Comment by talkorem** [comment]
I'm converting this to a discussion.

**Comment by earlylifemicro-lab** [comment]
Thank you for your useful reply.
